pkg/deployment/reconcile/plan_builder_utils.go

```diff
@@ -60,8 +60,8 @@ func emptyPlanBuilder(ctx context.Context,
 
 func removeConditionActionV2(actionReason string, conditionType api.ConditionType) api.Action {
 	return api.NewAction(api.ActionTypeSetConditionV2, api.ServerGroupUnknown, "", actionReason).
-		AddParam(setConditionActionV2KeyAction, setConditionActionV2KeyTypeRemove).
-		AddParam(setConditionActionV2KeyType, string(conditionType))
+		AddParam(setConditionActionV2KeyAction, string(conditionType)).
```

Comment:
why contitiond type is added to action, and action is added to type?

Reply:
Type is type of action taken within Update action, Action is the ConditionType